scripts/run_e2e_tests.py

```diff
@@ -0,0 +1,472 @@
+
+GECKO_PROVIDER_BAK_FILE_PATH = os.path.join(
+    WEBDRIVER_PROVIDER_PATH, 'geckodriver.js.bak')
+
+WEBPACK_BIN_PATH = os.path.join(
+    common.CURR_DIR, 'node_modules', 'webpack', 'bin', 'webpack.js')
+PATTERN_FOR_REPLACE_WEBDRIVER_CODE = r'this\.osArch = os\.arch\(\);'
+PROTRACTOR_CONFIG_FILE_PATH = os.path.join(
+    'core', 'tests', 'protractor.conf.js')
+BROWSER_STACK_CONFIG_FILE_PATH = os.path.join(
+    'core', 'tests', 'protractor-browserstack.conf.js')
+HASHES_FILE_PATH = os.path.join('assets', 'hashes.json')
+
+_PARSER = argparse.ArgumentParser(description="""
+Run this script from the oppia root folder:
+   bash scripts/run_e2e_tests.sh
+
+The root folder MUST be named 'oppia'.
+
+
+  --suite=suite_name Performs test for different suites, here suites are the
+        name of the test files present in core/tests/protractor_desktop/ and
+        core/test/protractor/ dirs. e.g. for the file
+        core/tests/protractor/accessibility.js use --suite=accessibility.
+        For performing a full test, no argument is required.
+Note: You can replace 'it' with 'fit' or 'describe' with 'fdescribe' to run a
+single test or test suite.
+""")
+
+_PARSER.add_argument(
+    '--browserstack',
+    help='Run the tests on browserstack using the'
+         'protractor-browserstack.conf.js file.',
+    action='store_true')
+_PARSER.add_argument(
+    '--skip-install',
+    help='If true, skips installing dependencies. The default value is false.',
+    action='store_true')
+_PARSER.add_argument(
+    '--sharding', default=True, type=bool,
+    help='Disables/Enables parallelization of protractor tests.'
+         'Sharding must be disabled (either by passing in false to --sharding'
+         ' or 1 to --sharding-instances) if running any tests in isolation'
+         ' (fit or fdescribe).',
+    )
+_PARSER.add_argument(
+    '--sharding-instances', type=int, default=3,
+    help='Sets the number of parallel browsers to open while sharding.'
+         'Sharding must be disabled (either by passing in false to --sharding'
+         ' or 1 to --sharding-instances) if running any tests in isolation'
+         ' (fit or fdescribe).')
+_PARSER.add_argument(
+    '--prod_env',
+    help='Run the tests in prod mode. Static resources are served from'
+         ' build directory and use cache slugs.',
+    action='store_true')
+_PARSER.add_argument(
+    '--community_dashboard_enabled', action='store_true',
+    help='Run the test after enabling the community dashboard page.')
+
+_PARSER.add_argument(
+    '--suite', default='full',
+    help='Performs test for different suites, here suites are the'
+         'name of the test files present in core/tests/protractor_desktop/ and'
+         'core/test/protractor/ dirs. e.g. for the file'
+         'core/tests/protractor/accessibility.js use --suite=accessibility.'
+         'For performing a full test, no argument is required.')
+
+SUBPROCESSES = []
+
+
+def check_screenshot():
```

Comment:
From reading the function, sounds like a better name would be something like ensure_screenshots_dir_is_removed().

Reply:
Updated